pkgs/development/python-modules/dtaidistance/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, llvmPackages
+, cython
+, numpy
+}:
+
+buildPythonPackage rec {
+  pname = "dtaidistance";
+  version = "2.3.10";
+
+  src = fetchFromGitHub {
+    owner = "wannesm";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-JP/EljFemywgRgspxFzIoHeTtI8gK1SbKgBcGTmzmqw=";
+  };
+
+  nativeBuildInputs = [ cython numpy ];
+
+  buildInputs = [ llvmPackages.openmp ];
```

```suggestion
  buildInputs = [ llvmPackages.openmp ];
  
  propagatedBuildInputs = [ numpy ];
```
https://github.com/wannesm/dtaidistance/blob/d914ab85021f67ff1c58d45727e0e4844ad26d8e/setup.py#L459